### ISSUE TITLE
Casing in new API: IsFullwidth vs IsFullHeight

### ISSUE THREAD

**Issue by alfonsogarciacaro** [MangelMaxime/Fulma#106]
A small inconsistency in the new API that may confuse some users: [IsFullwidth](https://github.com/MangelMaxime/Fulma/search?utf8=%E2%9C%93&q=IsFullwidth&type=) appears in several places in the new API. However, for the Hero layout, `IsFullHeight` (Height in upper case) is used (and `IsHalfHeight`).

**Comment by MangelMaxime** [comment]
I am much in favor for `IsFullWidth`, I find it easier to read and follow the `PascalCase` convention for the DUs. I was trying to follow the same naming convention as Bulma but probably not a good idea and it's time to take a decision :).

`IsFullwidth` have always been present in several place I believe. And I don't think we will be able to prevent that. I tried by the past to use `interface` like we do for `Fable.Helpers.React.Props`, but the result wasn't really successful.

**Comment by rmunn** [comment]
I agree that having `IsFullHeight` and `IsFullwidth` would be confusing. But since Bulma has `is-fullwidth` rather than `is-full-width`, it would also be confusing to *not* have `IsFullwidth` available as a name. I suggest having both: the default spelling would be `IsFullWidth`, and the alternate spelling of `IsFullwidth` would have an extra comment `// Alternate spelling for consistency with Bulma's is-fullwidth class`.

**Comment by MangelMaxime** [comment]
Quote from @evilz 

> Alternate spelling make Api more complex for nothing.
> New user will ask them self what is the difference ^^ and then there is none.
> 
> My point of view ;p

I kind of agree with him and I more and more see Fulma as an "intelligent wrapper" around Bulma. What I mean, is there is already some API which has been added to make Fulma usage easier.

For example we have an `NoColor` case.
This allow:

```fs
// I need to use list comprehension because I need to not yield Input.Color in some case
Input.text [ yield Input.Value formData.Name 
             yield Input.Disabled false 
             yield Input.Props [ OnChange (getValue >> ChangeName >> dispatch) ] 
             if hasError "name" formData.Errors then 
              yield Input.Color IsDanger ]

// This is an helpers because it's used at several places in my code
let inputColor key errors =
    if hasError key errors then
        IsDanger
    else
        NoColor
    |> Input.Color

Input.text [ Input.Value formData.Name 
             Input.Disabled false
             Input.Props [ OnChange (getValue >> ChangeName >> dispatch) ]
             inputColor "name" formData.Errors ]
```

This is the most recent addition to Elmish, but there is other example like that especially in the Fulma.Extensions module.

My idea being Fulma, is to improve the user experience as much as possible, if we follow this idea then we should remove the `alternate spelling` thing.

**Comment by Nhowka** [comment]
Probably the user will have the documentation as an entry point, being from Fulma or Bulma side. There they will see it being used as `IsFullWidth` and `is-fullwidth` respectively. They will test the one that matches it more closely and it will work.

In a HMR environment, if the user gets curious about the difference it can be tested in less than 15 seconds. Both will work and if curiosity gets too high, there will be these issues explaining why they exist.

So I see no harm in having both ways just for casing consistency. Having `IsFullHeight` next to `IsFullwidth` is way more stressing to some people than having `IsFullWidth` next to `IsFullwidth` hidden in code completion.
